cmd/argocd/commands/project.go

```diff
@@ -78,8 +78,9 @@ func (opts *projectOpts) GetSignatureKeys() []v1alpha1.SignatureKey {
 // NewProjectCommand returns a new instance of an `argocd proj` command
 func NewProjectCommand(clientOpts *argocdclient.ClientOptions) *cobra.Command {
 	var command = &cobra.Command{
-		Use:   "proj",
-		Short: "Manage projects",
+		Use:     "proj",
+		Short:   "Manage projects",
+		Aliases: []string{"projs", "project", "projects"},
```

Comment:
Adding aliases is a good improvement! The only concern is to have it consistently in all commands.

Can you please remove aliases in this PR and add them in a separate PR for all commands?

Reply:
Sure, I added this because I found myself typing 'project' instead of 'proj' a few times. Do you want me to move all the aliases into a new PR or just the project ones (and then add any other commands that could use them too)?

Reply:
I think we should add aliases to all commands that manage entities ( like `app`, `proj`, `account`, `cluster`, `repo`, `repocreds`, `cert`  ).

Reply:
ok, i created an issue to follow up, and removed this change from this PR

Reply:
thank you!